server/i18n/en.json

```
@@ -7184,7 +7184,7 @@
   },
   {
     "id": "app.user.update_active.user_limit.exceeded",
-    "translation": "Can't activate user. Server exceeds safe user limit. Contact your system administrator. Error code: ERROR_SAFETY_LIMITS_EXCEEDED."
+    "translation": "Can't activate user. Server exceeds safe user limit. Contact administrator with: ERROR_SAFETY_LIMITS_EXCEEDED."
```

Assuming you want these to be the same, there's still a slight difference
```suggestion
    "translation": "Can't activate user. Server exceeds safe user limit. Contact your administrator with: ERROR_SAFETY_LIMITS_EXCEEDED."
```